core/src/com/unciv/logic/civilization/ExploredRegion.kt

```diff
@@ -12,7 +12,7 @@ import com.unciv.ui.components.tilegroups.TileGroupMap
 import kotlin.math.abs
 import kotlin.math.sqrt
 
-class ExploredRegion () : IsPartOfGameInfoSerialization {
```

Comment:
is the default costructor neccessary for serialization? Should I suppress the warning?

Reply:
thanks!